pyproject.toml

```diff
@@ -54,15 +54,11 @@ relative_files = true
 line-length = 120
 exclude = "(.eggs|.git|.hg|.mypy_cache|.venv|_build|buck-out|build|dist)"
 
-[tool.isort]
-known_first_party = [
-    "torchmetrics",
-    "integrations",
-    "unittests",
-]
-skip_glob = []
-profile = "black"
-line_length = 120
```

Comment:
Why is this not needed anymore?

Reply:
this is not related to this PR so I'll drop it as a special commit but as we merged #1797 we do not use this any more